charts/console/values.yaml

```diff
@@ -90,6 +90,21 @@ config:
     password: ""
     username: ""
 
+  ## @param config.conduktorSql.enabled Enable SQL database for Conduktor Console
+  ## @param config.conduktorSql.host Your Conduktor Console SQL Database host
+  ## @param config.conduktorSql.port Your Conduktor Console SQL Database port
+  ## @param config.conduktorSql.name Your Conduktor Console SQL Database name
+  ## @param config.conduktorSql.username Your Conduktor Console SQL Database username
+  ## @param config.conduktorSql.password Your Conduktor Console SQL Database password
+  ## NOTE: This is a Beta Feature still early in development
+  conduktorSql:
+    host: ""
+    port: 5432
+    name: "conduktor-sql"
+    password: ""
+    username: ""
+    enabled: false
```

Comment:
should be 
```yaml
config: 
  kafkasql:
    database: 
      host: ""
      port: 5432
      name: "conduktor-sql"
      password: ""
      username: ""
```
to match console config and allow configuration of kafkasql other properties like `commitOffsetEveryInSec`  `refreshTopicConfigurationEveryInSec` `cleanExpiredRecordEveryInHour`  and `consumerGroupId`